### PR TITLE
owner test: make use of helper

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/tests/owners.rs [hunk 1]
```diff
@@ -93,12 +93,8 @@ fn new_crate_owner() {
     token.enqueue_publish(crate_to_publish).good();
 
     // Add the second user as an owner (with a different case to make sure that works)
-    let user2 = app.db_new_user("Bar");
-    token.add_user_owner("foo_owner", "BAR");
```
Comment (Turbo87) [inline src/tests/owners.rs:97]: this appears to be testing the case where the invitation is using uppercase while the username is using a different style. if we went with this refactoring then I think we would lose that check, no?

Comment (tshepang) [reply src/tests/owners.rs:97]: agreed... is it fine if I uppercase the user on the helper (and add the case comment)

**Comment by Turbo87** [comment]
@tshepang I still have a hard time figuring out what the advantage of this PR is. is it fixing any real issues? the tests appear to work fine the way they are, so I'm wondering where the need to change the code came from.

**Comment by tshepang** [comment]
I was reading tests and noticed that the test fn doesn't use the helper fn, so am here reducing code duplication.

Note that I am so attached, so if this doesn't add enough value, I would be ok closing it.

**Comment by Turbo87** [comment]
I think I'd prefer to keep the test as is, and only use the helper for the other tests. always having the helper use uppercase feels a bit suboptimal to me and since the helper ultimately only saves us 3 lines of code I'm not sure it's worth it to deduplicate here. 

thanks for the PR though! :)

**Pull request closed without merging**